lib/serve-file-middleware.js

```diff
@@ -14,9 +14,9 @@ function createStaticFileMiddleware(logger, injector) {
 		if(req.url.indexOf(STATIC_PREFIX) !== 0) {
 			return next();
 		}
-		
-		let file = `${TMP_STATIC_FILES_DIR}/${req._parsedUrl.pathname.substr(STATIC_PREFIX.length)}`;
-		
+
+		let file = `${TMP_STATIC_FILES_DIR}/${decodeURI(req._parsedUrl.pathname.substr(STATIC_PREFIX.length))}`;
```

Comment:
Doesn’t the middleware automatically decode URI escapes? I would expect a field named `_parsedUrl` to be, well, parsed…